repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/valkyrie/CullingWMethodRestrictionTest.groovy

```diff
@@ -253,7 +253,7 @@ class CullingWMethodRestrictionTest extends ReposeValveTest {
         where:
         method    | tenantID       | deviceID | deviceID2 | permission     | responseCode | handlings
         "HEAD"    | randomTenant() | "520708" | "511123"  | "view_product" | "200"        | 1
-        "CONNECT" | randomTenant() | "520707" | "520706"  | "view_product" | "400"        | 0
+        "CONNECT" | randomTenant() | "520707" | "520706"  | "view_product" | "200"        | 1
```

Comment:
Another behavior change. We used to check that a request method was supported, rejecting the request if not. In this particular case, we do not support `CONNECT`. Which we probably shouldn't, since the intent of `CONNECT` is to open a TCP tunnel with the origin service, bypassing processing performed by Repose at the application layer.

See: https://github.com/rackerlabs/repose/blob/master/repose-aggregator/core/repose-core/src/main/java/org/openrepose/powerfilter/PowerFilter.java#L415-L417